packages/typespec-azure-resource-manager/lib/models.tsp

```diff
@@ -302,6 +302,25 @@ model EntityTagProperty {
   eTag?: string;
 }
 
+/**
+ * Model used only to spread in the standard `etag` envelope property for a resource
+ *
+ * @example
+ *
+ * ```typespec
+ * model Foo is TrackedResource<FooProperties> {
+ *   // Only have standard Succeeded, Failed, Cancelled states
+ *   ...ETagProperty;
+ * }
+ * ```
+ */
+@doc("The etag property envelope.")
+model ETagProperty {
```

Comment:
Worried here this will be easily used instead of the other one, this feels like a more intuitive name.

Is that for legacy scenario only? Do we really need a template just that that?